bin/vaultclient

```diff
@@ -73,25 +73,26 @@ function getAdminConfig(configFilePath) {
     return readConfigFile(configFilePath);
 }
 
-function action(cmd, fn, args) {
+function action(cmd, fn, _, options) {
+    const args = options.optsWithGlobals();
```

Comment:
Maybe worth a small comment to explain why we have an unused variable.
Also I think we should also check that `options` is set first, before calling `optsWithGlobals` unless this is guaranteed by commander? Before, we were checking for the previous `args`: we should do the same here